src/GDGUkraine/rest_controller.py

```diff
@@ -511,6 +511,23 @@ def generate_invites(self, id):
             raise HTTPError(500, "Cannot save generated invites") from e
         return {"ok": True}
 
+    @cherrypy.tools.json_out()
+    @cherrypy.tools.authorize()
+    def record_visit(self, id):
+        '''POST /api/events/:id/check-in'''
+        req = cherrypy.request
+        orm_session = req.orm_session
+        reg_id = int(id)
+        reg_data = api.get_event_registration_by_id(orm_session, reg_id)
+        if not reg_data:
+            raise HTTPError(400,
```

Comment:
We receive `reg_id` from url. Thus someone (with sufficient privileges) can easily misspell id in the URL (like `1111` instead of `111`) and get a `Bad request error`. However the actual reason is `Not found`.